tests/test_test.py

```diff
@@ -0,0 +1,78 @@
+from pathlib import Path
+from _pytest.capture import CaptureFixture
+from _pytest.monkeypatch import MonkeyPatch
+
+import rasa.model
+import rasa.cli.utils
+
+
+def monkeypatch_get_latest_model(tmp_path: Path, monkeypatch: MonkeyPatch):
+
+    latest_model = tmp_path / "my_test_model.tar.gz"
+
+    def mock_get_latest_model():
+        return str(latest_model)
+
+    monkeypatch.setattr(rasa.model, "get_latest_model", mock_get_latest_model)
```

Comment:
optional (whatever you prefer)
```suggestion
    monkeypatch.setattr(rasa.model, "get_latest_model", lambda: str(latest_model))
```